cython/setup.py.in

```diff
@@ -5,13 +5,13 @@ try:
 except ImportError:
     from distutils.core import setup, find_packages
 
-if 'SETUP_PY_NO_CHECK' not in os.environ:
-    script_path = os.path.abspath(os.path.realpath(__file__))
-    install_path = os.path.abspath(os.path.realpath(os.path.join('${GTSAM_CYTHON_INSTALL_PATH}${GTSAM_BUILD_TAG}', 'setup.py')))
-    if script_path != install_path:
-        print('setup.py is being run from an unexpected location: "{}"'.format(script_path))
-        print('please run `make install` and run the script from there')
-        sys.exit(1)
+# if 'SETUP_PY_NO_CHECK' not in os.environ:
```

Comment:
What is this commented out code?

Reply:
It is to check whether `setup.py` is being ran in the correct directory. I think this is unnecessary and is not common practice...

Reply:
Should we just kill it then? @thduynguyen any advice?

Reply:
I'll kill it :)

Reply:
NUKED